nuttx-configs/aerofc-v1/nsh/defconfig

```diff
@@ -536,9 +536,9 @@ CONFIG_STM32_I2CTIMEOTICKS=10
 #
 # Configuration
 #
-CONFIG_STM32_BBSRAM=y
-CONFIG_STM32_BBSRAM_FILES=4
-CONFIG_STM32_SAVE_CRASHDUMP=y
+# CONFIG_STM32_BBSRAM is not set
+# CONFIG_STM32_BBSRAM_FILES is not set
+# CONFIG_STM32_SAVE_CRASHDUMP is not set
 # CONFIG_STM32_HAVE_RTC_COUNTER is not set
 # CONFIG_STM32_HAVE_RTC_SUBSECONDS is not set
```

Comment:
Saving to BBSRAM is working but as it will never mount a SDCard and call "hardfault_log reset" (line 63 of rcS) the reboot counter will increase until it get stuck.
As saving to BBSRAM is only useful if saving in SDCard I think is better remove it from AeroFC.